content/kubermatic/master/guides/monitoring_logging_alerting/user_cluster/admin_guide/_index.en.md

```diff
@@ -0,0 +1,301 @@
+- Create configmap with `grafana-dashboards` as the name prefix. You can add multiple such configmaps with your Dashboards json data. For example:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: grafana-dashboards-example
+  namespace: mla
+data:
+  example-dashboard.json: <your-dashboard-json-data>
+```
+
+- Add dashboards to the User Cluster MLA Grafana Helm chart values. You can add more Dashboards directly in the [`dashboards` in the value.yaml file](https://github.com/kubermatic/mla/blob/main/config/grafana/values.yaml#L41).
+
+After the new Dashboards are applied to the Seed Cluster, they will be available across all Grafana Organizations, and they can be found in the Grafana UI under Dashboards -> Manage.
+
+## Debugging
+
+This chapter describes some potential problems that you may face in a KKP installation and the steps you can take to resolve then.
+
+**Prometheus / Loki datasource for an user cluster is not available in the Grafana UI:**
+
+- Make sure you are switched to the proper Grafana Organization (see the “Switch between Grafana Organizations” section of this documentation)
+- Make sure that user cluster Monitoring / Logging is enabled for the user cluster (In KKP UI, you should see green checkboxes on the Cluster Page):
+
+![MLA UI - Cluster View](/img/kubermatic/master/monitoring/user_cluster/ui_cluster_view.png)
+
+**Metrics / Logs are not available in Grafana UI for some user cluster:**
+
+- Make sure that User Cluster Monitoring / Logging is enabled for the user cluster (In KKP UI, you should see green checkboxes on the Cluster Page):
+
+![MLA UI - Cluster View](/img/kubermatic/master/monitoring/user_cluster/ui_cluster_view.png)
+
+- Check that Prometheus / Promtail was deployed an is running in the user cluster:
+
+```bash
+kubectl get pods -n mla-system
+NAME                          READY   STATUS    RESTARTS   AGE
+prometheus-68f7485456-jj7v6   1/1     Running   0          11m
+promtail-cm4qd                1/1     Running   0          6m11s
+```
+
+- Check the logs of Prometheus / Promtail pods
+- Check that the MLA Gateway pod is running in the user cluster namespace in the Seed cluster:
+
+```bash
+kubectl get pods -n cluster-cxfmstjqkw | grep mla-gateway
+mla-gateway-6dd8c68d67-knmq7                  1/1     Running   0          22m
+```
+
+- Check the logs of the MLA Gateway pods
+- Check the status of the pods in the `mla` namespace in the seed cluster:
+
+```bash
+kubectl get pods -n mla
+```
+
+- If there are any pods crashing, review their logs for the root cause
+
+## Uninstallation
+
+**WARNING:** Before proceeding with any of the following steps, make sure that you backup all data that you may still need - metrics data / logs in the object store, alertmanager / rules configuration, Grafana dashboards.
```

Comment:
```suggestion
{{% notice warning %}} 
Before proceeding with any of the following steps, make sure that you backup all data that you may still need - metrics data / logs in the object store, alertmanager / rules configuration, Grafana dashboards.
{{% /notice %}}
```
This will look like the warning block at the top of this doc: https://docs.kubermatic.com/kubermatic/v2.17/guides/addons/kubeflow/